spec/services/stats/r003_business_unit_performance_report_spec.rb

```diff
@@ -296,8 +296,7 @@ module Stats
       describe '#results' do
         before do
           Timecop.freeze Time.new(2017, 6, 30, 12, 0, 0) do
-            # report = R003BusinessUnitPerformanceReport.new
-            report = R003BusinessUnitPerformanceReport.new(period_start: Date.today.beginning_of_year, period_end: Date.today, generate_bu_columns: true)
+            report = R003BusinessUnitPerformanceReport.new(period_start: Date.new(2017, 6, 2), period_end: Date.today, generate_bu_columns: true)
```

Comment:
Why does this need to be 'today'? - I always prefer to have defined dates in tests as it makes them more deterministic.

Reply:
@willmcb, for this one,  the time has been freeze at ` Timecop.freeze Time.new(2017, 6, 30, 12, 0, 0)`, so the today won't be changed, just a short cut for using `2017, 6, 30, 12, 0, 0`